rts/Game/UnsyncedGameCommands.cpp

```diff
@@ -3631,7 +3629,6 @@ void UnsyncedGameCommands::AddDefaultActionExecutors()
 	AddActionExecutor(AllocActionExecutor<ChatActionExecutor>("Spec", "s:", true));
 
 	AddActionExecutor(AllocActionExecutor<TrackActionExecutor>());
-	AddActionExecutor(AllocActionExecutor<TrackOffActionExecutor>());
```

Comment:
Is this action redundant with some other one?

Reply:
Ah, I see this: `// TODO merge together with "TrackOff" to "Track 0|1", and deprecate the two old ones`

Reply:
How does the `Track` argument look like?